Makefile

```diff
@@ -43,3 +43,7 @@ check-env:
 ifndef PROJECT_ID
 	$(error PROJECT_ID is undefined)
 endif
+
+ifndef ZONE
+	$(error ZONE is undefined)
```

Comment:
I'd actually vote against this addition FWIW. I think it's better to provide a default that can be overwritten instead of requiring input for variables like this

But that's just me, feel free to merge either way

Reply:
@Daniel-Sanche on line 17 a default is provided for ZONE and 18 for CLUSTER. I can only imagine these checks are put here just in case a user removes them? 🤷‍♂️

Reply:
previously the only check was only for PROJECT_ID, which is the one required variable we can't set a default for.

Yeah I guess this new check doesn't really change anything though unless the user removes the default, so I'm fine either way